client/js/global.js

```diff
@@ -41,6 +42,20 @@ function fmOverlay() {
   }
 
   /**
+   * Set Overlay cookies.
+   */
+  function setCookie() {
+    const name = cookieName;
```

Comment:
Totally can & will do; forgot to go back and refactor this after moving the scope of `cookieName` to the parent function. Thanks for catching this